learning_resources_search/views.py

```diff
@@ -98,6 +99,53 @@ def get(self, request):
             return Response(errors, status=400)
 
 
+@method_decorator(blocked_ip_exempt, name="dispatch")
+@extend_schema_view(
+    get=extend_schema(
+        parameters=[LearningResourcesSearchRequestSerializer()],
+        responses=LearningResourcesVectorSearchResponseSerializer(),
+    ),
+)
+@action(methods=["GET"], detail=False, name="Search Learning Resources")
+class LearningResourcesVectorSearchView(ESView):
+    """
+    Vector Search for learning resources
+    """
+
+    permission_classes = ()
+
+    @method_decorator(
+        cache_page_for_anonymous_users(
+            settings.SEARCH_PAGE_CACHE_DURATION, cache="redis", key_prefix="search"
+        )
+    )
+    @extend_schema(summary="Vector Search")
+    def get(self, request):
+        request_data = LearningResourcesSearchRequestSerializer(data=request.GET)
```

Comment:
Also, can this be a V0 api for now? The V1 apis are supposed to be stable and have good documentation and be usable by an outside project. It can be moved to V1 once we build it out more.